susemanager/rhn-conf/rhn_server_susemanager.conf

```diff
@@ -28,3 +28,9 @@ temp_token_lifetime = 60
 token_lifetime = 525600
 # automatically deploy tokens once they are refreshed
 token_refresh_auto_deploy = true
+
+# when re-generating bootstrap repos remove old content first
+bootstrap_repo_flush = 1
+
+# data module used for mgr-create-bootstrap-repo
+bootstrap_repo_datamodule = mgr_bootstrap_data
```

Comment:
Is this because we plan to support more data files :-?

Reply:
In case a customer must change the package list he can write his own file and configure the tool to use it. Our `mgr_bootstrap_data.py` file is not a config, it is a data file which will be overwritten with every update.

I think we will not prominent announce this, but it is there when needed .